frontend/src/App.vue

```diff
@@ -8,7 +8,8 @@ import { useSocketStore } from './stores/socketStore';
 
 if (useUserStore().isLoggedIn) {
 	useUserStore().refreshMe();
-	useSocketStore().initializeOnline();
+	const socketStore = useSocketStore();
+	socketStore.initialize();
```

Comment:
Why did you declare a socketstore first? Does this have any effect on availability of the data? (Cause it's still a big frustration with the game hihi)

Reply:
that was the hope, but I think it didn't do anything
so I also call initialize (if needed) before creating the DM..

the actual "solve" was that i added a delay before posting the new channel from the frontend... I have it 1 extra second, just to make sure the socket store had some time to get started... not sure if that would also help in your case...